web/html/src/manager/content-management/list-filters/list-filters.tsx

```diff
@@ -134,11 +135,11 @@ const ListFilters = (props: Props) => {
     </div>
   );
 
-  const unusedFilter = <Button className="btn-link" handler={onSelectUnused} text={t("Select unused")}></Button>;
+  const unusedFilter = <Button className="btn-default" handler={onSelectUnused} text={t("Select unused")}></Button>;
 
   const deleteSelected = (
     <Button
-      className={`${selectedIdentifiers.length ? "btn-danger" : "btn-disabled"}`}
+      className={`${selectedIdentifiers.length ? "btn-danger" : "btn-danger disabled"}`}
```

Comment:
I think the standalone class `disabled` should be unnecessary here, since we already set and unset the `disabled` attribute?